### PR TITLE
Feature/add code base

### PR DESCRIPTION
**Pull request by assadounto**
### In this feature 

- [ ] I added Web-pack-Config

- [ ]  I set-up jest unit test

### CODE REVIEW

**Comment by teckim** [comment]
# Well Done @assadounto 👏🏼 
The codebase looks great for our project, but here is what I think you can improve:

- Kindly add the dependencies, either by installing them or by writing them in the `package.json` file.
- What do you think if we remove all the `eslint-disable ...` and try to fix the linter issues.

> _Please check the comments_

File: package.json [hunk 1]
```diff
@@ -16,5 +16,16 @@
   "bugs": {
     "url": "https://github.com/teckim/tv-maze/issues"
   },
-  "homepage": "https://github.com/teckim/tv-maze#readme"
+  "homepage": "https://github.com/teckim/tv-maze#readme",
+  "devDependencies": {
```
Comment (teckim) [inline package.json:25]: We can add our dependencies here.

**Review by teckim — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (assadounto) [reply package.json:25]: okay

Comment (assadounto) [reply package.json:25]: okay done

**Pull request merged**